fmriprep/interfaces/images.py

```diff
@@ -134,28 +154,12 @@ def _run_interface(self, runtime):
 {discard_list}
 \t\t</ul>
 """
-DISCARD_TEMPLATE = """\t\t\t\t<li><abbr title="{path}">{basename}</abbr></li>"""
-
-
-class ConformSeriesInputSpec(BaseInterfaceInputSpec):
-    t1w_list = InputMultiPath(File(exists=True), mandatory=True,
-                              desc='input T1w images')
-    max_scale = traits.Float(3.0, usedefault=True,
-                             desc='Maximum scaling factor in images to accept')
 
+DISCARD_TEMPLATE = """\t\t\t\t<li><abbr title="{path}">{basename}</abbr></li>"""
 
-class ConformSeriesOutputSpec(TraitedSpec):
-    t1w_list = OutputMultiPath(exists=True, desc='conformed T1w images')
-    out_report = File(exists=True, desc='conformation report')
-
-
-class ConformSeries(SimpleInterface):
-    """Conform a series of T1w images to enable merging.
-
-    Performs two basic functions:
 
-    #. Orient to RAS (left-right, posterior-anterior, inferior-superior)
-    #. Along each dimension, resample to minimum voxel size, maximum number of voxels
+class TemplateDimensions(SimpleInterface):
+    """Filter a series of T1w images based on the requirements for up-sampling.
```

Comment:
Proposed replacement for this first line, focusing on the main goal of this interface:

```
Finds template target dimensions for a series of T1w images, filtering low-resolution images, if necessary.

Along each axis, the minimum voxel size (zoom) and the maximum number of voxels (shape) are found
across images.
```